Gopkg.toml

```diff
@@ -35,15 +35,15 @@ required = [
   branch = "master"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```

Comment:
This is to be reverted once https://github.com/giantswarm/apprclient/pull/26 is merged.